packages/core/src/browser/keyboard/browser-keyboard-layout-provider.spec.ts

```diff
@@ -51,7 +51,7 @@ describe('browser keyboard layout provider', function (): void {
         const container = new Container();
         container.bind(BrowserKeyboardLayoutProvider).toSelf();
         container.bind(ILogger).to(MockLogger);
-        container.bind(LocalStorageService).toSelf().inSingletonScope();
```

Comment:
please revert it, it is done that extenders could rebind `LocalStorageService` if they want